project/src/main/java/com/google/sps/data/errors/FetcherException.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data.errors;
+/**
+ * FetcherException represents an error returned by the PlacesFetcher.
+ */
+public class FetcherException extends Exception {
```

Comment:
Should this class be final?

Reply:
I thought about it but I had an idea that when we have a better error handeling design this class will have descendants that treat specific fetching errors. Though probably as long is we don't have that design it should be final.